pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -647,3 +668,10 @@ internal class PageLayoutInfo {
         return "PageLayoutInfo(page = $page, layoutSize=$layoutSize)"
     }
 }
+
+internal fun Int.floorMod(other: Int): Int {
```

Comment:
Add a comment explaining what this is doing. Also, this can be private.

Reply:
`floorMod` is also used in `Pager.kt`. So this can't be private.
Any good idea?

And I will add a comment. 🙇

Reply:
Sounds good.